drivers/fuel_gauge/bq27z746/bq27z746.c

```diff
@@ -307,7 +291,7 @@ static int bq27z746_init(const struct device *dev)
 }
 
 static const struct fuel_gauge_driver_api bq27z746_driver_api = {
-	.get_property = &bq27z746_get_props,
+	.get_property = &bq27z746_get_prop,
 	.set_property = &bq27z746_set_props,
```

Comment:
doing get and set separately?

Reply:
That was the plan. But I can combine them into a single PR if you'd like - normally I try to do 1 commit = 1 PR when reasonable. I should've called that out more explicitly in the top comment.

Reply:
the change is probably symmetric, I think it make sense to do both at once, up to you tough

Reply:
I'll do both at once then!